packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -326,6 +318,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   @autobind
   protected onSuggestionClick(ev: React.MouseEvent<HTMLElement>, item: any, index: number) {
     this.addItemByIndex(index);
+    this.setState({ suggestionsVisible: false });
   }
```

Comment:
When the suggestion is clicked, don't you need to preventDefault/stopPropagation on the event?